src/scss/base/_base.scss

```
@@ -1,16 +1,200 @@
-// Base styles
+* {
+  outline: transparent;
+}
 
 *,
-*::before,
-*::after {
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+html {
   margin: 0;
   padding: 0;
+  font-family: 游ゴシック体, 'Yu Gothic', YuGothic, 'ヒラギノ角ゴシック Pro', 'Hiragino Kaku Gothic Pro', メイリオ, Meiryo, Osaka, 'ＭＳ Ｐゴシック', 'MS PGothic', sans-serif;
+  font-weight: normal;
+  color: #000000;
+  font-size: 14px;
+  line-height: 1.6;
+  background: #eaf1f4;
+}
+
+* {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
   box-sizing: border-box;
 }
 
-html,
 body {
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  scroll-behavior: smooth;
+  margin: 0px;
+  position: relative;
+}
+
+h1,
+h2,
+h3,
+h4 {
+  font-weight: normal;
+  margin: 0px;
+}
+
+h2 {
+  line-height: 1.3;
+}
+
+ul {
+  margin: 0px;
+  padding: 0px;
+
+  li {
+    list-style: none;
+  }
+}
+
+.clearfix:after {
+  content: " ";
+  /* Older browser do not support empty content */
+  visibility: hidden;
+  display: block;
+  height: 0;
+  clear: both;
+}
+
+.container {
+  width: 100%;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.ctn-inner {
+  max-width: 1000px;
+  margin: 0 auto;
+}
+
+.flex {
+  display: -webkit-flex;
+  display: -moz-flex;
+  display: -ms-flex;
+  display: -o-flex;
+  display: flex;
+}
+
+.justyify-center {
+  justify-content: center;
+}
+
+a {
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: none;
+  }
+
+  &:focus {
+    outline: none;
+    text-decoration: none;
+  }
+
+}
+
+/* general setting
+-----------------------------------------------------------*/
+dl {
+  dt {
+    margin-bottom: 6px;
+  }
+
+  dd {
+    margin-bottom: 26px;
+  }
+}
+
+pre {
+  border-radius: 2px;
+  margin-bottom: 33px;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+
+  li {
+    list-style: none;
+  }
+}
+
+p {
+  margin: 0px;
+
+  &:last-child {
+    margin-bottom: 0px;
+  }
+}
+
+a:hover,
+a:link,
+a:visited,
+a:active {
+  text-decoration: none;
+}
+
+img {
+  width: auto;
+  max-width: 100%;
+  height: auto;
+  vertical-align: middle;
+}
+
+figure {
+  padding: 0;
+  margin: 0;
+}
+
+button,
+html input[type="button"],
+input[type="reset"],
+input[type="submit"] {
+  -webkit-appearance: button;
+  cursor: pointer;
 }
+
+input {
+  height: auto;
+}
+
+:not(output):-moz-ui-invalid {
+  box-shadow: none;
+}
+
+:not(output):-moz-ui-invalid:-moz-focusring {
+  box-shadow: none;
+}
+
+output:-moz-ui-invalid {
+  color: none;
+}
+
+select {
+  outline: none;
+  -webkit-transition: all .3s;
+  transition: all .3s;
+  resize: none;
+  padding: 6px;
+  border-radius: 2px;
+}
+
+.sp {
+  display: none !important;
+}
+
+@media screen and (max-width: 768px) {
```

Please define variable for screen responsive. like
`$breakpoints: ( xs: 'screen and (max-width: 768px)', sm: 'screen and (max-width: 959px)' ) !default`
and create mixins: 
```suggestion
@mixin media-breakpoint($breakpointName)
  $mediaQuery: map_get($breakpoints, $breakpointName)
  @if ($mediaQuery != null)
    @media #{$mediaQuery}
      @content
```

when use want to use responsive. you can use `@include media-breakpoint('xs')` instead of `@media screen and (max-width: 768px)`